src/ui/atoms/button/button.scss

```diff
@@ -40,14 +36,42 @@ $background-colors: (
 
     &.size-small {
       padding: 4px 10px;
+
+      &.icon {
+        padding: 7px;
+      }
     }
 
     &.size-medium {
       padding: 6px 16px;
+
+      &.icon {
+        padding: 10px;
+      }
     }
 
     &.size-large {
       padding: 8px 22px;
+
+      &.icon {
+        padding: 17px;
+      }
+    }
+
+    &.icon {
+      min-height: unset;
+      min-width: unset;
+      max-width: unset;
+      font-size: 0;
+      border-radius: 50%;
+    }
+
+    .okp4-button-label {
+      display: grid;
+      grid-auto-flow: column;
+      gap: 10px;
```

Comment:
I will have added a little more gap between elements to better fit the wireframes (the goal is also not to have as much space as in the wireframe):
![image](https://user-images.githubusercontent.com/47554752/181252245-4fd7ce59-2d0e-4065-a5e7-3ed4c027c8a4.png)

Reply:
I could add some more gap and if the user wants to add more spacing they can just increase the width and position it where they want :)